freqtrade/freqai/RL/Base5ActionRLEnv.py

```diff
@@ -49,6 +49,7 @@ def step(self, action: int):
         self._update_unrealized_total_profit()
         step_reward = self.calculate_reward(action)
         self.total_reward += step_reward
+        self.tensorboard_metrics[self.actions._member_names_[action]] += 1
```

Comment:
Why would you prefer to have this here instead of inside the below `is_tradesignal()` logic?

While the action count *before* is_tradesignal is absolutely valuable, I am not sure it is *more* valuable than knowing the true trade signal action counts. Now the user will see many actions that were never acted upon due to the is_tradesignal protection.

Reply:
Never mind, you can ignore this comment. Maybe we add another entry to the dictionary that is "acted upon" counts or something.